samples/AspNetIntegration/FooMiddleware.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Middleware;
+
+namespace AspNetIntegration
+{
+    internal class FooMiddleware : IFunctionsWorkerMiddleware
```

Comment:
Can we use a better name here?

Reply:
Actually, for a simple example, is a middleware required? If this sample is primarily to show the ASP.NET Core integration, we should keep it simpler and move this to a different sample.

Reply:
I was thinking we would have multiple examples in this app (including the simple trigger). Should we have separate apps? At least for the middleware? Or do we want to leave that out of these samples in the repo and leave it for the docs?